src/Controls/src/Core/TabbedPage/TabbedPage.Windows.cs

```diff
@@ -165,7 +165,10 @@ void OnApplyTemplateFinished(object? sender, EventArgs e)
 		void OnNavigationViewSizeChanged(object sender, SizeChangedEventArgs e)
 		{
 			if (_navigationView != null)
+			{
+				this.InvalidateMeasure();
```

Comment:
If you remove the call to `this.Arrange` and leave this.InvalidateMeasure does this all still work?

The call to arrange inside the SizeChanged method here doesn't seem correct. 

Also, can you add a test?

Reply:
@PureWeen, I have implemented the suggested changes, tested the fix across all basic scenarios, and also ran UI tests.

**Testcase**: The test case cannot be created for this scenario as resizing the window to a minimized state is not feasible during testing. Therefore, we have not added a test case for this scenario. Could you suggest alternative ways to address this, since resizing the window to a minimized state isn't possible during testing.

Reply:
Do you need to resize the Window or minimize and maximize it?

Reply:
@jsuarezruiz, The issue occurs only when resizing the window. Could you please share your suggestions on how to resolve it.